### DISCUSSION TITLE
issue with image prompt in Colabratory

### DISCUSSION THREAD

**Discussion by dwhubber** [lllyasviel/Fooocus#1208] [General]
Very often an image prompt causes the Google Colab to error out and gradio stops working when doing an image prompt.

Wondering how I can negate this or what I'm doing wrong in the Colab?

[Fooocus] Encoding positive #2 ...
[Fooocus] Encoding negative #1 ...
[Fooocus] Encoding negative #2 ...
[Fooocus] Image processing ...
Requested to load CLIPVisionModelWithProjection
Loading 1 new model
^C

**Comment by dwhubber** [comment]
This is using realistic mode with an image loaded as either a prompt or face swap, quality high, seed 1234

**Comment by turbodizayn** [comment]
Hi, I have issue often((( Allways when combining inpaintong and image prompt, for example!
![issue_2023-11-16_233339803](https://github.com/lllyasviel/Fooocus/assets/97593553/3514202e-03b7-428e-9348-b89a20df312c)

**Comment by dwhubber** [comment]
Interesting for me it combining of the faceswap and image prompt

**Comment by dwhubber** [comment]
Similar thing just happened today again

Just errors out

100% 104M/104M [00:06<00:00, 16.1MB/s]
Downloading: "https://github.com/xinntao/facexlib/releases/download/v0.2.2/parsing_parsenet.pth" to /content/Fooocus/models/controlnet/parsing_parsenet.pth

100% 81.4M/81.4M [00:24<00:00, 3.54MB/s]
^C

**Comment by mashb1t** [comment]
please check https://github.com/lllyasviel/Fooocus/issues/1377 or https://github.com/lllyasviel/Fooocus/issues/423 for a potential solution.

**Comment by dwhubber** [comment]
Thanks @mashb1t. 

For anyone else seeing this it was indeed caused by the free collab version and going to pro version of collab with

`!python entry_with_update.py --share --preset realistic --always-high-vram --all-in-fp16` from @AlanTuring1978 works.
